forge-gui/res/cardsfolder/upcoming/green_suns_twilight.txt

```diff
@@ -0,0 +1,12 @@
+Name:Green Sun's Twilight
+ManaCost:X G
+Types:Sorcery
+A:SP$ DigMultiple | DigNum$ SVar$X/Plus.1 | Reveal$ True | ChangeValid$ Creature,Land | ChangeLater$ True | Optional$ True | RememberChanged$ True | ImprintRest$ True | SubAbility$ DBBranch | SpellDescription$ Reveal the top X plus one cards of your library. Choose a creature card and/or a land card from among them. Put those cards into your hand and the rest on the bottom of your library in a random order. If X is 5 or more, instead put the chosen cards onto the battlefield or into your hand and the rest on the bottom of your library in a random order.
```

Comment:
I don't think it's Optional$ but min=1 & max=2?

Reply:
Hmm, that would mean almost every DigMultiple card needs refactoring?

Reply:
How come? all others have "may"

Reply:
I think I have this sorted...